modules/rostests/kmtests/ntos_mm/NtCreateSection_drv.c

```diff
@@ -413,17 +414,38 @@ TestIrpHandler(
     {
         Fcb = IoStack->FileObject->FsContext;
         ok(Fcb != NULL, "Null pointer!\n");
-        if (IoStack->FileObject->SectionObjectPointer != NULL &&
-            IoStack->FileObject->SectionObjectPointer->SharedCacheMap != NULL)
+
+        if (IoStack->FileObject->SectionObjectPointer == NULL)
+        {
+            ok(!CcIsFileCached(IoStack->FileObject), "File is cached\n");
+
+            trace("No cache to clean up\n");
+        }
+        else if (IoStack->FileObject->SectionObjectPointer->SharedCacheMap == NULL)
         {
             LARGE_INTEGER Zero = RTL_CONSTANT_LARGE_INTEGER(0LL);
 
+            ok(!CcIsFileCached(IoStack->FileObject), "File is cached\n");
+
+            // CcUninitializeCacheMap must always be called,
+            // even when CcInitializeCacheMap has not been called.
+            trace("CcUninitializeCacheMap(Zero)\n");
+            KeInitializeEvent(&CacheUninitEvent.Event, NotificationEvent, FALSE);
+            CcUninitializeCacheMap(IoStack->FileObject, &Zero, &CacheUninitEvent);
```

Comment:
It don't see the point of passing 0 here, as the file cache isn't initialized.

Reply:
Passing zero means the file got truncated to 0 and cache needs to be purged to that point. That's no point here as cache wasn't initialized.

Reply:
> I'll try to retest that...

5-month old ReactOS code:
Rebuilt `NULL`, fails: [WTB 40631](https://testbot.winehq.org/JobDetails.pl?Key=40631)
Reran `Zero`, succeeds: [WTB 40633](https://testbot.winehq.org/JobDetails.pl?Key=40633)

> That's no point here as cache wasn't initialized.

I have no idea what Windows does internally, but these are the test results!
Or do you mean there is a separate issue in ReactOS (km)test code? Maybe fixed in the meantime?

For what it's worth, see also my question in the Jira issue.